src/main/webapp/resources/scripts/services/EventService.coffee

```diff
@@ -0,0 +1,44 @@
+angular.module('calendar').service 'EventService', ($http) ->
+  new class
+    baseUrl = '/event/'
+
+    constructor: ->
+
+    getBetweenDate: (startDate, endDate) ->
+      console.log(startDate)
+      console.log(endDate)
+
+      #create fake data
+      date = new Date()
+      d = date.getDate()
+      m = date.getMonth()
+      y = date.getFullYear()
+      return [
```

Comment:
in coffee you can simply write json like this:
[
        name: 'event1'
        baseData:
          startDate: new Date(y, m, d - 3, 16, 0)
          endDate: new Date(y, m, d - 3, 23, 0)
      ,
        name: 'event2'
        baseData:
          startDate: new Date(y, m, d, 12, 0)
          endDate: new Date(y, m, d, 18, 30)
      , (...)

by default returned function value is the result of the last line - in this case 'return' word may be ommited